source/framework/include/marlin/Application.h

```diff
@@ -142,6 +143,11 @@ namespace marlin {
      */
     RandomSeedManager &randomSeedManager() ;
 
+    /**
+     *  @brief Get book store manager
+     */
+    BookStoreManager &bookStoreManager() const ;
```

Comment:
const returns a const reference, else no const returns a reference. Don't mix the two, make it clear.

Reply:
changed return type to  `const std::unique_ptr<BookStoreManager>&` because the Applications owns the BookStoreManager, but the Manager can change, also when the Application is const.